docs/Changelog.md

```diff
@@ -13,6 +13,7 @@ Nussknacker versions
 * More precise TypeInformation generation
     * [#1338](https://github.com/TouK/nussknacker/pull/1338) Defining TypeInformation based on TypingResult
     * [#1343](https://github.com/TouK/nussknacker/pull/1343) Aggregators compute stored types    
+    * [#1343](https://github.com/TouK/nussknacker/pull/1359) Improvements variable output validation 
```

Comment:
..typo

mprovements **in** variable output validation 
or 
Improve variable output validation